src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 2.2.4
 ++++++
+* `vmss create`: reserve large enough frontend port range for overprovision
```

Comment:
maybe we can start the change with "now reserves a...". 

Also not sure if we should end with "... for overprovision" or "... for overprovisioning".